specification/paths/UpdateShipmentStatus.json

```diff
@@ -0,0 +1,77 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "shipments.manage"
+        ]
+      }
+    ],
+    "summary": "Manually update a shipment's status.",
+    "description": "This endpoint allows a user to manually update a shipment's status.",
+    "requestBody": {
+      "description": "The shipment id to be updated and the status details to update it with.",
+      "required": true,
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "properties": {
+              "data": {
+                "type": "object",
+                "required": [
+                  "shipment_id",
+                  "status_id"
+                ],
+                "properties": {
+                  "shipment_id": {
+                    "type": "string",
+                    "format": "uuid",
+                    "example": "ea002ef2-0c0f-432a-8263-c8dcb191e728"
+                  },
+                  "status_id": {
+                    "type": "string",
+                    "format": "uuid",
+                    "example": "be1c7cb5-e8d4-494a-9637-edb27ecab7c1"
+                  },
+                  "carrier_status": {
+                    "$ref": "#/components/schemas/CarrierStatus"
+                  },
+                  "error": {
+                    "$ref": "#/components/schemas/Error"
```

Comment:
Do we want to post an error? It's not mentioned in the story.

Reply:
Yeah, I forgot to discuss this with you. Let's discuss when you get in today =)